src/Components/Search/Search.test.js

```diff
@@ -1,4 +1,34 @@
 import React from 'react'
 import '@testing-library/jest-dom'
-import { screen, render, waitFor } from '@testing-library/react'
-import Search from './Search.js'
\ No newline at end of file
+import { screen, render, waitFor, fireEvent } from '@testing-library/react'
+import Search from './Search.js'
+jest.mock('../../API');
```

Comment:
Will we need to mock the API requests to test the search component?

Reply:
We could also create a mock store to hold a book to search for. Then test that the book is being displayed?